tests/plugins/test_koji_delegate.py

```diff
@@ -0,0 +1,153 @@
+"""
+Copyright (c) 2015 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+from __future__ import unicode_literals, absolute_import
+
+import os
+from copy import deepcopy
+from textwrap import dedent
+
+try:
+    import koji as koji
+except ImportError:
+    import inspect
+    import sys
+
+    # Find our mocked koji module
+    import tests.koji as koji
+    mock_koji_path = os.path.dirname(inspect.getfile(koji.ClientSession))
+    if mock_koji_path not in sys.path:
+        sys.path.append(os.path.dirname(mock_koji_path))
+
+    # Now load it properly, the same way the plugin will
+    del koji
+    import koji as koji
+
+from atomic_reactor.plugins.pre_koji_delegate import KojiDelegatePlugin
+from atomic_reactor.plugins.pre_check_and_set_rebuild import CheckAndSetRebuildPlugin
+from atomic_reactor.plugins.pre_reactor_config import (ReactorConfigPlugin,
+                                                       WORKSPACE_CONF_KEY,
+                                                       ReactorConfig)
+from flexmock import flexmock
```

Comment:
Import order, 3rd party modules should be before project modules

Reply:
ok